packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -327,7 +338,7 @@ export class StripeHelper {
     idempotencyKey: string
   ): Promise<Stripe.Customer> {
     try {
-      return await this.stripe.customers.create(
+      const stripeCustomer = await this.stripe.customers.create(
```

Comment:
Do we use these flows anymore?

Reply:
Hmm, I don't think so - must be a spot I missed in #5850. I think we just use `createPlainCustomer` now

Reply:
I can remove the method if it is safe to @lmorchard